docs/faqs/seq-or-val.md

```diff
@@ -4,7 +4,7 @@ A: No, no, and no!
 
 An Arbitrum Chain's Sequencer(s) and Validators and completely distinct entities, with their own distinct roles.
 
-The Sequencer is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), the Sequencer has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reorder transactions or _temporarily_ delay a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.
+The [Sequencer](sequencer) is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), it has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reorder transactions or _temporarily_ delay a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.
```

Comment:
```suggestion
The [Sequencer](../sequencer) is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), it has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reorder transactions or _temporarily_ delay a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.
```

Reply:
How do these links works in Docusaurus? Can we have `.md` here so it works in both GitHub and developer website?

Reply:
yep, it appears we can; I'll do a sweep at the end to change them all to relative file links